warehouse/integrations/bigquery/bigquery.go

```diff
@@ -264,6 +271,18 @@ func (bq *BigQuery) deduplicationQuery(tableName string, columnMap model.TableSc
 		partitionColumn, partitionType = bq.partitionColumn(), bq.partitionType()
 	)
 
+	// if partitioningInWarehouse is not nil, then we use the partition column and type from the warehouse
+	if partitioningInWarehouse != nil {
+		if partitionColumn != partitioningInWarehouse.Field {
+			bq.logger.Warnn("partition column mismatch in config and warehouse",
+				logger.NewStringField("partitionColumnInConfig", partitionColumn),
+				logger.NewStringField("partitionColumnInWarehouse", partitioningInWarehouse.Field),
+			)
+		}
+		partitionColumn = partitioningInWarehouse.Field
```

Comment:
Does Field come as UPPER case?

Reply:
Field is nothing but a column name, column names are not case-sensitive in Bigquery.